Gemfile.lock

```diff
@@ -343,36 +354,38 @@ GEM
       method_source (~> 0.8.1)
       slop (~> 3.4)
     racc (1.4.14)
-    rack (1.6.4)
-    rack-dev-mark (0.7.4)
+    rack (2.0.1)
+    rack-dev-mark (0.7.5)
       rack (>= 1.1)
     rack-test (0.6.3)
       rack (>= 1.0)
-    rails (4.2.6)
-      actionmailer (= 4.2.6)
-      actionpack (= 4.2.6)
-      actionview (= 4.2.6)
-      activejob (= 4.2.6)
-      activemodel (= 4.2.6)
-      activerecord (= 4.2.6)
-      activesupport (= 4.2.6)
+    rails (5.0.0)
```

Comment:
__[brakeman]__
* Rails 5.0.0 content_tag does not escape double quotes in attribute values (CVE-2016-6316). Upgrade to 5.0.0 [:ref](https://groups.google.com/d/msg/ruby-security-ann/8B2iV2tPRSE/JkjCJkSoCgAJ)

<!-- Comment by [SideCI](https://sideci.com) -->